sdk/python/jobs/automl-standalone-jobs/automl-forecasting-recipes-univariate/automl-forecasting-recipe-univariate-run.ipynb

```diff
@@ -0,0 +1,1503 @@
+    "    instance_count=2,\n",
+    "    max_concurrency_per_instance=2,\n",
+    "    mini_batch_size=10,\n",
+    "    output_action=BatchDeploymentOutputAction.APPEND_ROW,\n",
+    "    output_file_name=output_file,\n",
+    "    retry_settings=BatchRetrySettings(max_retries=3, timeout=30),\n",
+    "    logging_level=\"info\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, start a model deployment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1670412553908
+    }
+   },
+   "outputs": [],
+   "source": [
+    "ml_client.begin_create_or_update(batch_deployment).wait()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We need to create the Input, representing URI folder, because the batch endpoint is intended to process multiple files at a time. In this example we will use only one test file, which we have uploaded to the blob storage earlier. This file must be available through the url link.\n",
+    "\n",
+    "#### Create an inference job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1670412560499
+    }
+   },
+   "outputs": [],
+   "source": [
+    "job = ml_client.batch_endpoints.invoke(\n",
+    "    endpoint_name=batch_endpoint_name,\n",
+    "    input=my_test_data_input,\n",
+    "    deployment_name=\"univariate-non-mlflow-deployment\",  # name is required as default deployment is not set\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will stream the job output to monitor the execution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1670412706051
+    }
+   },
+   "outputs": [],
+   "source": [
+    "job_name = job.name\n",
+    "batch_job = ml_client.jobs.get(name=job_name)\n",
+    "print(batch_job.status)\n",
+    "# stream the job logs\n",
+    "ml_client.jobs.stream(name=job_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Download the prediction result for metrics calculation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The output of forecast output is saved in JSON format. You can use it to calculate test set metrics and plot predictions and actuals over time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1670412708663
+    }
+   },
+   "outputs": [],
+   "source": [
+    "ml_client.jobs.download(job_name, download_path=\".\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1670412709310
+    }
+   },
+   "outputs": [],
+   "source": [
+    "pred_df = pd.read_json(output_file, orient=\"table\")\n",
```

Comment:
Let us use the CSV file. I guess the problem here is that datetimes were incorrectly serialized in JSON format. Let us use CSV instead.